api/src/app/modules/attachment/attachment.service.ts

```diff
@@ -160,8 +160,10 @@ export class AttachmentService extends DataService<Attachment, Repository<Attach
 
   async findFileDatabase(id: number, user?: User): Promise<AttachmentFileResponse> {
     // Works, but fileContents being treated as a Buffer...
-    const entity:Attachment = await this.repository.findOne(id, this.addCommonRelationsToFindOptions(
-      { select: [ 'id', 'projectId', 'fileName', 'attachmentType' ] }));
+    const revisedOptions = this.addCommonRelationsToFindOptions(this.addCommonRelationsToFindOptions(
```

Comment:
Would be good to comment why this is necessary.

Reply:
I added comment on this in other feature branch.
I will now close this pr in favor of more stable upgraded Angular(15)/NestJS(v9) branch but leave this branch still alive in case you need this.